README.md

```diff
@@ -108,6 +108,10 @@ spec:
 
 ## Release Notes
 
+### 2.9.1
+
++ fixed problem with using full verification instead of verification by key fields to matching filter in sequence rule when order check is disabled
```

Comment:
```suggestion
+ fixed a problem while using full verification instead of verification by key fields, to matching the filter in sequence rule when order check is disabled
```

Reply:
Why `matching the filter`? I think that `matching filter` here refers to a filter which is used for matching. If that so I think that `to matching filter in` should be changed to `in matching-filter of`

Reply:
If "matching filter" refers to a specific filter, then I agree with you that should be changed to **in matching filter of**.